tests/Doctrine/Tests/Common/Annotations/DocLexerTest.php

```diff
@@ -308,4 +308,26 @@ public function testTokenAdjacency() : void
         self::assertFalse($lexer->nextTokenIsAdjacent());
         self::assertFalse($lexer->moveNext());
     }
+
+    public function testMarkerAnnotationLocaleTr() : void
+    {
+        setlocale(LC_ALL, 'tr_TR.utf8', 'tr_TR');
```

Comment:
Are we resetting the locale after this test?

Reply:
Add reset locale after this test

Reply:
Either reset the locale in a `tearDown` method or use `finally`: a failing assertion will leave the locale changed to `tr_TR`.

Reply:
fixed